packages/api/cms-api/src/user-permissions/user-permissions.types.ts

```diff
@@ -21,6 +21,7 @@ export type PermissionsForUser =
 export type ContentScopesForUser = ContentScope[] | UserPermissions.allContentScopes;
 
 export interface AccessControlServiceInterface {
+    isAllowed(user: CurrentUserInterface, permission: keyof Permission): boolean;
     canAccessScope(requestScope: ContentScope, user: CurrentUserInterface): boolean;
```

Comment:
Should canAccessScope and isAllowed be better aligned?

- user as first vs second argument
- is vs can
- Scope in method name vs Permission not in method name

Reply:
I'd love to!
34fcbf24dd747db7a8e27f4e3e2d59228d89475a
- Rename canAccessScope to isAllowedContentScope and move to AccessControlService
- Rename isAllowed to isAllowedPermission
- Use term contentScope more consistently